### DISCUSSION TITLE
v1.2

### DISCUSSION THREAD

**Discussion by tianpf** [spatial-go/geoos#96] [General]
- Added dynamic buffer algorithm, which can support buffer algorithms with different widths for lines.
- Optimized the union algorithm and realized union support for non convex polygon.
- Solved the Issue issue.

<hr /><em>This discussion was created from the release <a href='https://github.com/spatial-go/geoos/releases/tag/v.1.2'>v1.2</a>.</em>

**Comment by philhug** [comment]
v1.2 was tagged as `v.1.2` instead of `v.1.2`, this prevents `go mod` from picking up the latest version.
You might want to retag the release.
